web/api/alert.go

```diff
@@ -39,4 +46,4 @@ func (s AlertManagerService) addAlerts(w http.ResponseWriter, r *http.Request, p
 	}
 
 	s.Manager.Receive(alerts)
-}
+}
```

Comment:
Please configure your editor to end files with newlines (http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline).